File: Examples/CSharp/AsposePDF/Forms/FormFieldFont14.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Text;

namespace CSharp.AsposePdf.Forms
{
    public class FormFieldFont14
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Forms();

            // Open document
            Document pdfDocument = new Document(dataDir + "FormFieldFont14.pdf");

            // Get particular form field from document
            Aspose.Pdf.InteractiveFeatures.Forms.Field field = pdfDocument.Form["textbox1"] as Aspose.Pdf.InteractiveFeatures.Forms.Field;

            // Create font object
            Aspose.Pdf.Text.Font font = FontRepository.FindFont("ComicSansMS");

            // Set the font information for form field
            field.DefaultAppearance = new Aspose.Pdf.InteractiveFeatures.DefaultAppearance(font, 10, System.Drawing.Color.Black);

            // Save updated document
            pdfDocument.Save(dataDir + "FormFieldFont14_out.pdf");
        }
    }
}

File: Examples/VisualBasic/AsposePDF/Forms/FormFieldFont14.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf.Forms
Imports Aspose.Pdf
Imports Aspose.Pdf.Text
Namespace AsposePDF.Forms
    Public Class FormFieldFont14
        Public Shared Sub Run()
            ' ExStart:FormFieldFont14
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Forms()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("FormFieldFont14.pdf"))

            ' Get particular form field from document
            Dim field As Field = TryCast(pdfDocument.Form("textbox1"), Field)

            ' Create font object
            Dim font As Font = FontRepository.FindFont("ComicSansMS")

            ' Set the font information for form field
            ' Field.DefaultAppearance = new Aspose.Pdf.Forms.in.DefaultAppearance(font, 10, System.Drawing.Color.Black);

            dataDir = dataDir & Convert.ToString("FormFieldFont14_out_.pdf")
            ' Save updated document
            pdfDocument.Save(dataDir)
            ' ExEnd:FormFieldFont14
            Console.WriteLine(Convert.ToString(vbLf & "Form field font setup successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace
